src/app/route/CategoryPage/CategoryPage.container.js

```diff
@@ -306,9 +315,9 @@ export class CategoryPageContainer extends PureComponent {
         });
     }
 
-    _requestCategoryProductsInfo() {
+    _requestCategoryProductsInfo(customFilters = {}) {
```

Comment:
Use getter within a class instead of passing argument

Reply:
Unfortunately, it is not possible since different sources might be used for "customFilters" variable - see line 177 and line 335.